workbench/src/main/main.js

```diff
@@ -80,6 +80,7 @@ export const createWindow = async () => {
 
   const investExe = findInvestBinaries(ELECTRON_DEV_MODE);
   settingsStore.set('investExe', investExe);
+  settingsStore.set('mamba', findMambaExecutable(ELECTRON_DEV_MODE));
```

Comment:
This is nitpicky but do we want to be consistent with whether we create a variable for a returned binary path or just call it from within the function signature?